src/OrchardCore/OrchardCore.Abstractions/Shell/Removing/IShellRemovingManager.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Environment.Shell.Removing;
+
+/// <summary>
+/// Host service managing the removing of a given tenant.
+/// </summary>
+public interface IShellRemovingManager
```

Comment:
```suggestion
public interface IShellRemovalManager
```

Reply:
Okay, will change it, I hesitated at first ;)

Reply:
Do you want to also rename all `...RemovingHandler` to `...RemovalHandler` ?

The project folders `Removing` to `Removal` ?

Reply:
For now only done for `IShellRemovalManager` and then `ShellRemovalManager`.

Reply:
The handler is good I think, since we're talking about an event raised while the process of removal is happening (i.e. "removing" is good).

Reply:
Okay I got it